src/openforms/appointments/contrib/qmatic/tests/test_plugin.py

```diff
@@ -1,6 +1,7 @@
-from datetime import date, datetime, timezone
+from datetime import date, datetime
```

Comment:
Was timezone an unused import?

Reply:
NO, it was a wrong import. It is replaced with `django.utils.timezone`